### ISSUE TITLE
Bug: Internal server error when accessing /moderator/presentations

### ISSUE THREAD

**Issue by TimKardol** [HZ-HBO-ICT/it-conference#275]
When accessing https://weareinittogether.nl/moderator/presentations and no presentations are pending it throws a 500 status code. I am not able to reproduce it on the local environment. More investigation needed.

**Comment by TimKardol** [comment]
It seems I cannot reproduce the problem on the branches `release` and `development`

**Comment by TimKardol** [comment]
```
[previous exception] [object] (ErrorException(code: 0): Attempt to read property \"is_approved\" on null at /mnt/web308/c1/75/512066875/htdocs/site_2023-09-29_140649/app/Models/Presentation.php:141)
[stacktrace]
```

This is the error the log produced.

**Comment by TimKardol** [comment]
Closed in #280
